go.sum

```diff
@@ -1665,8 +1666,8 @@ github.com/quasilyte/go-consistent v0.0.0-20190521200055-c6f3937de18c/go.mod h1:
 github.com/quasilyte/go-ruleguard v0.2.0/go.mod h1:2RT/tf0Ce0UDj5y243iWKosQogJd8+1G3Rs2fxmlYnw=
 github.com/quasilyte/regex/syntax v0.0.0-20200407221936-30656e2c4a95/go.mod h1:rlzQ04UMyJXu/aOvhd8qT+hvDrFpiwqp8MRXDY9szc0=
 github.com/rcrowley/go-metrics v0.0.0-20181016184325-3113b8401b8a/go.mod h1:bCqnVzQkZxMG4s8nGwiZ5l3QUCyqpo9Y+/ZMZ9VjZe4=
-github.com/redhatinsights/yggdrasil v0.0.0-20220210193750-d696ee7a54bb h1:J+WMR8ee4Yrkg7nCqXOaE8oPyKbhdc7AbVSokEwnKMA=
-github.com/redhatinsights/yggdrasil v0.0.0-20220210193750-d696ee7a54bb/go.mod h1:pzYHLc9dg7BTax4FlcgVjdrN7SWHrmXCaQ2kUXWUGK8=
+github.com/redhatinsights/yggdrasil v0.0.0-20220309180936-d91759a7f332 h1:VPcx9axsKg7kifVVNcpF4qhfdc8BSQCrrFdfTdZJ0OE=
```

Comment:
Do we know when yggdrasil will be release with the changes?

Reply:
No timeline yet, but with these changes and also with all HTTP transport ones. I'll investigate :-)